tox.ini

```diff
@@ -0,0 +1,56 @@
+[tox]
+envlist =
+    check-{style,build}
+    test{,-warnings,-cov}-xdist
```

Comment:
Don't you also have the specify the `py*` stuff here?

Reply:
Also the specs here don't match actual tags you use in the workflows.

Reply:
good catch!